modules/serverless-channel-default.adoc

```diff
@@ -2,11 +2,13 @@
 //
 //  * serverless/event_workflows/serverless-channels.adoc
 
-[id="serverless-inmemorychannel_{context}"]
-= Using the default development channel configuration
+[id="serverless-channel-default_{context}"]
+= Configuring the default channel type
 
-When you install Knative Eventing, the following `default-ch-webhook` config map is created automatically in the `knative-eventing` namespace:
+When you install Knative Eventing, the `default-ch-webhook` config map is created automatically in the `knative-eventing` namespace.
```

Comment:
The config map is managed by the Operator, so its content must be configured in the KnativeEventing CR per https://issues.redhat.com/browse/SRVKE-621